test/lib/jdk/test/lib/artifacts/ArtifactResolver.java

```diff
@@ -68,6 +71,37 @@ public static Path resolve(String name, Map<String, Object> artifactDescription,
         return  manager.resolve(name, artifactDescription, unpack);
     }
 
+    /**
+     * Retrieve an artifact/library/file from a repository or local file system.
+     * <p>
+     * Artifacts are defined with the {@link jdk.test.lib.artifacts.Artifact}
+     * annotation. The file name should have the format ARTIFACT_NAME-VERSION.EXT
```

Comment:
It could also be a directory, right? So maybe `path name` is more precise.

Also, you haven't defined what each piece in ARTIFACT_NAME-VERSION.EXT is. On the other hand, the names in `Artifact` are `name`, `revision`, and `extension`.

Reply:
This sentence doesn't really belong here anyway so I removed it entirely